packages/types/src/errors/grid_client/kyc.ts

```diff
@@ -0,0 +1,53 @@
+import { BaseError } from "../base_error";
+import { ErrorModules } from "../modules";
+
+export enum Errors {
+  Unverified,
+  InvalidResponse,
+  RateLimit,
+  badRequest,
```

Comment:
Why the inconsistency? Change to BadRequest, Unauthorized!

Reply:
updated; thank you!